framework/audio/midi/miditypes.h

```diff
@@ -50,10 +52,10 @@ enum CntrType {
 };
 
 struct Event {
-    uint32_t tick{ 0 };
-    EventType type{ ME_INVALID };
-    int a{ 0 };
-    int b{ 0 };
+    uint32_t tick = 0;
+    EventType type = ME_INVALID;
+    int a = 0;
+    int b = 0;

```

Comment:
Can you explain what these variables mean?

Reply:
Each midi event has a tick when to apply it, an event type and two arguments. Depending on the type of event, the arguments have a different meaning. For example, for the event type NOTE_ON, this is `noteNumber` and `velocity`

https://www.mixagesoftware.com/en/midikit/help/HTML/midi_events.html